src/DuffelPayments/Refunds/Refunds.ts

```diff
@@ -0,0 +1,33 @@
+import { Resource } from '../../Resource'
+import { DuffelResponse } from '../../types'
+import { CreateRefund, Refund } from './RefundsType'
+
+export class Refunds extends Resource {
+  /**
+   * Endpoint path
+   */
+  path: string
+
+  constructor(args: any) {
+    super(args)
+    this.path = 'payments/refunds'
+  }
+
+  /**
+   * You should use this API to get the complete, up-to-date information about a Refund.
+   * @param {string} id - Duffel's unique identifier for the Refund
+   * @link https://duffel.com/docs/api/refunds/get-refund-by-id
+   */
+  public get = async (id: string): Promise<DuffelResponse<Refund>> =>
+    this.request({ method: 'GET', path: `${this.path}/${id}` })
+
+  /**
+   * Create a Refund to refund some money to a customer that they paid using a Payment Intent. You must specify the amount and currency. The currency is currently limited to the currency in which the Payment Intent was made.
+   * @param {Object.<CreateFund>} params - Endpoint params (amount, currency and payment_intent_id)
+   */
+  public create = async (
+    params: CreateRefund
+  ): Promise<DuffelResponse<any>> => {
```

Comment:
This seems to come with a response right? Should we type it accordingly? 
<img width="871" alt="image" src="https://user-images.githubusercontent.com/6373626/183104773-a232bc36-05cb-4046-a060-6b00c06f3a30.png">

Reply:
Ah, I missed this one! I put any just in the beginning and forgot about it. Thanks for noticing! Will add a fix shortly.

Reply:
Done!